inspirehep/migrator/tasks.py

```diff
@@ -361,17 +360,16 @@ def migrate_record_from_mirror(
     except ValidationError as exc:
         path = ".".join(exc.schema_path)
         LOGGER.warn(
-            f"Migrator Validator Error: {path}, Value: %r, Record: %r",
+            "Migrator Validator Error: %r, Value: %r, Record: %r",
+            path,
             exc.instance,
             prod_record.recid,
         )
         prod_record.error = exc
         db.session.merge(prod_record)
     except PIDValueError as exc:
         message = f"pid_type:'{exc.pid_type}', pid_value:'{exc.pid_value}'"
-        LOGGER.error(
-            f"{exc}: %s, Record: %r", message, prod_record.recid, exc_info=True
-        )
+        LOGGER.exception("(%s), Record: %r", message, prod_record.recid)
```

Comment:
It was done like this on purpose so it would aggregate different exceptions in separately. Anyway, if you change it I would also add the exception type somewhere (this was the intent, but it should have been `type(exc)` instead of `exc`).

Reply:
LOGGER.exception is taking care of that